helm/mds/tests/deployment-agency-override_test.yaml

```diff
@@ -111,21 +103,13 @@ tests:
           path: spec.template.spec.containers[0].ports[0].protocol
           value: TCP
         documentIndex: 0
-      - equal:
-          path: spec.template.spec.containers[0].volumeMounts[0].name
-          value: mds-config-files
-        documentIndex: 0
-      - equal:
-          path: spec.template.spec.containers[0].volumeMounts[0].mountPath
-          value: /mds-config/
-        documentIndex: 0
       - equal:
           path: spec.template.spec.containers[0].env[0].name
           value: PORT
         documentIndex: 0
       - equal:
           path: spec.template.spec.containers[0].env[0].value
-          value: "4000"
+          value: '4000'
```

Comment:
All of these `yaml` files probably used `""` instead of `''` for a reason, may be problematic to change them Cc. @jwtodd @invertigo

Reply:
I'm not sure how those changes got in there. Reverting.